src/cockpit/samples.py

```diff
@@ -228,12 +228,12 @@ def sample(self, samples: Samples) -> None:
                 num_sectors_read = fields[5]
                 num_sectors_written = fields[9]
 
-                # ignore device-mapper and md
-                if dev_major in ['9', '253']:
+                # ignore mdraid
+                if dev_major == '9':
                     continue
 
-                # Skip partitions
-                if dev_name[:2] in ['sd', 'hd', 'vd'] and dev_name[-1].isdigit():
+                # Skip partitions and device mapper
+                if dev_name[:2] in ['sd', 'hd', 'vd', 'md'] and dev_name[-1].isdigit():
```

Comment:
"dm"?

Reply:
I think the devicemapper check should be `dev_name[:3] == "dm-"`, just to be clearer.